### PR TITLE
feat : sort by latest video

### PR DESCRIPTION
**Pull request by ragini07**
Added sort by newest and oldest videos

### CODE REVIEW

File: src/Utils/getDate.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const getDate = (createdAt) => {
+  console.log(createdAt);
```
Comment (manismk) [inline src/Utils/getDate.js]: can remove the console log

**Review by manismk — APPROVED** [review]
Clean code. Functionality works properly

File: src/Utils/getDate.js (hunk 1 above)
Comment (ragini07) [reply src/Utils/getDate.js]: done

**Review by Pratik1005 — APPROVED** [review]
sort is working, good to merge

**Pull request merged**